terraform/lib/dependabot/terraform/registry_client.rb

```diff
@@ -86,6 +86,20 @@ def service_url_for(service_key)
 
       attr_reader :hostname, :tokens
 
+      def get(endpoint:)
```

Comment:
Can't see any references to this method, could we ✂️  it?

Reply:
This was pulled in from a rebase off main. Not sure when it stopped being referenced but tests pass without it.